ufficio_soci/viste.py

```diff
@@ -663,6 +664,14 @@ def __semplifica_nome(nome):
             })
 
         persona_colonne = [y if y is not None else "" for y in [x(persona) for x in colonne]]
+        torna_utente = Persona.objects.get(codice_fiscale=persona_colonne[2])
```

Comment:
Questa vista (`us_elenco_download`) e' generica e viene utilizzata per lo scaricamento di tutti gli elenchi in Gaia. Se vuoi modificare un particolare elenco, devi modificare la classe Elenco rispettiva (in questo caso stai cercando `formazione.elenchi.ElencoPartecipantiCorsiBase`).